### DISCUSSION TITLE
MRFs Posting On Internet Website Availability Time Period

### DISCUSSION THREAD

**Discussion by ghost** [CMSgov/price-transparency-guide#69] [Q&A] [answered]
The rule calls out that the MRFs must be on a publicly available website
_"The final rules require non-grandfathered group health plans and health insurance issuers in the individual and group markets (plans and issuers) to disclose certain pricing information. Under the final rules a plan or issuer must disclose in-network negotiated rates, billed and out-of-network allowed, and in-network drug prices (including both "negotiated rates" and "historical net prices") through three machine-readable files posted on an internet website......made available to the public without restrictions that would impede the re-use of that information."_

Does this mean that mean the monthly MRFs are to be a
- cumulative posting of all files generated up to the current month, current and previous, to be publicly available on a internet website, or
- only the most current month's files would be available publicly on a internet website with the previous month's files removed from availability?

This is not a question based on any archival requirements of the MRFs, but what is to be publicly available on the internet website.

**Comment by shaselton-usds** [comment, marked as answer]
This is hopefully addressed on question 37 found in the [TiC websites' technical clarifications](https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification).
